crud/src/test/java/com/redhat/lightblue/crud/test/TestCRUDController.java

```diff
@@ -85,4 +86,10 @@ public MetadataListener getMetadataListener() {
     public void updatePredefinedFields(CRUDOperationContext ctx, JsonDoc doc) {
     }
 
+	@Override
+	public LightblueHealth checkHealth() {
```

Comment:
The checkHealth logic can be taken from https://github.com/lightblue-platform/lightblue-healthcheck.